javascript/stimulus_reflex.js

```diff
@@ -358,3 +375,5 @@ export default {
     debugging = !!value
   }
 }
+
+console.log('hop was here')
```

Comment:
Did you forget to remove this console.log?

Reply:
LOL sure did